README.md

```diff
@@ -2,34 +2,62 @@
 
 This repository contains smart contracts and front end interfaces for the Astria Shared Sequencer.
 
-### Rough-draft how to
+## Installation
 
-* Dependencies
-    * npm
-    * dotenv - `npm install -g dotenv-cli`
-    * Foundry + Forge - https://book.getfoundry.sh/getting-started/installation
+Make sure you have the following installed:
 
+  * npm
+  * dotenv - `npm install -g dotenv-cli`
+  * [Foundry + Forge](https://book.getfoundry.sh/getting-started/installation)
+
+Then, clone the repo using the `--recurse-submodules` flag to bring in all the submodules.
 ```bash
-# clone repo while also cloning submodules
 git clone --recurse-submodules git@github.com:astriaorg/astria-web3.git
+cd astria-web3
+```
+
+### Install dependencies
 
-# install dependencies
+```bash
 npm install -g dotenv-cli
 curl -L https://foundry.paradigm.xyz | bash
+```
+
+## Deployment
```

Comment:
Not all the things in this section are Deployments. Some of them are scripts or tests. I'm not sure we need this.

Reply:
removed this